Sources/NIO/Bootstrap.swift

```diff
@@ -966,4 +1133,134 @@ public final class NIOPipeBootstrap {
             }
         }
     }
+    
+    /// A channel option which can be applied to pipe bootstrap using shorthand notation.
+    /// - See: NIOPipeBootstrap.channelOptions(_ options: [Option])
+    public struct Option {
```

Comment:
Not sure what @Lukasa / @glbrntt think but should we move all the new stuff into a separate file? Like `UniversalBootstrapSupport.swift` we could have `ConvenienceOptionSupport.swift`?

Reply:
I think a separate file would be a good idea.

Reply:
Now all in separate file.